ddtrace/_logger.py

```diff
@@ -90,3 +90,11 @@ def _add_file_handler(
         logger.addHandler(ddtrace_file_handler)
         logger.debug("ddtrace logs will be routed to %s", log_path)
     return ddtrace_file_handler
+
+
+def set_log_formatting():
```

Comment:
do we need this? or should we only update L87 `ddtrace_file_handler.setFormatter(logging.Formatter(DD_LOG_FORMAT))` ?

since people usually configure the root logger and not our logger, we generally won't have handlers here, right?

Reply:
I didn't modify any of this code, I just moved it from: `ddtrace/bootstrap/sitecustomize.py`. I just think this module should encapsulate most our logging logic. 

I don't know why this logic was even introduced.  I don't think there's a lot of value in applying the `DD_LOG_FORMAT` to ddtrace loggers. 

It would be nice if we could introduce a `DD_TRACE_FORMAT_LOGGER` configuration that has a default value of `False` and when set to `True` formats customer's loggers to support trace-log injection (or it detects existing formatters and attempts to merge it with ours).